services/web/server/src/simcore_service_webserver/users/_users_rest.py

```diff
@@ -136,16 +164,16 @@ async def update_my_profile(request: web.Request) -> web.Response:
 _RESPONSE_MODEL_MINIMAL_POLICY["exclude_none"] = True
 
 
-@routes.get(f"/{API_VTAG}/users:search", name="search_users")
+@routes.get(f"/{API_VTAG}/admin/users:search", name="search_users_for_admin")
 @login_required
-@permission_required("user.users.*")
+@permission_required("user.admin.read")
```

Comment:
This should be accessible to POs

Reply:
it is. admin refers to "administration tasks" not admin role.